### DISCUSSION TITLE
Output formatet with backslashes, when presentet thru fastapi

### DISCUSSION THREAD

**Discussion by rhjensen79** [fastapi/fastapi#3111] [Questions] [answered]
Hoping somebody can point me in the right direction, since i'm 100% out of ideas. 

I'm pulling some data from MongoDB, using mongoengine.

If i run my job, without fastapi, i get a nicely formatted json output, which is what i want. 

If i run it thru Fastapi, it get's filled with \ and i can't figure out why.

The code i run looks like below :

@app.get("/box/country")
async def read_box_single(q: Optional[str] = None):
    boxes = Box.objects(country__icontains=q)
    boxes = boxes.to_json() 
    return {
        boxes
        }
        
And the result looks like this (part of it)
["[{\"_id\": {\"$oid\": \"6080121832fd10eecb454465\"}, \"name\"

If you want to see the full output, then it's public avaliable here : https://api.robert-jensen.dk/box/country?q=DK

Thanks in advance :-)

**Comment by tiangolo** [comment, marked as answer]
It would probably be better to ask in a GitHub issue, following the template that will guide you to structure the question in a way that makes it easier to understand your intention and need, to be able to help you better. And also providing a self-contained, [minimal, reproducible, example](https://stackoverflow.com/help/minimal-reproducible-example) that I can copy-paste to replicate it.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.

**Reply by rhjensen79** [reply]
Thanks for the answer. Honestly totaly forgot about it. 
I ended up solving it in a dfferent way. 
But THANKS for the reply. I really appreciate it :-) 
Got a lot more projects, with fastapi, so def not the last time, i will be asking :-) 

Thanks for a GREAT product.
